CGAT/CSV.py

```diff
@@ -44,7 +44,7 @@ def getMapColumn2Type(rows, ignore_empty=False, get_max_values=False):
                 if h not in max_values:
                     max_values[h] = int(row[h])
                 else:
-                    max_values[h] = max(h, int(row[h]))
```

Comment:
Thanks @TomSmithCGAT 

All looks good to me, but I would appreciate another pair of eyes looking at this line before merging.